eventually/task/models.py

```diff
@@ -42,16 +50,41 @@ class Task(models.Model):
     status = models.IntegerField(default=0, choices=STATUS_TYPE_CHOICES)
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
     updated_at = models.DateTimeField(auto_now=True)
+    event = models.ForeignKey(Event, on_delete=models.CASCADE, null=True)
+    users = models.ManyToManyField(CustomUser)
+
+    def __repr__(self):
+        """
+        Magic method that returns string representation of
+        task instance object.
+
+        :return: task title, task description, task status
+        """
 
+        return "{} {} {} {} {} {} {} {}".format(self.id,
+                                                self.title,
+                                                self.description,
+                                                self.status,
+                                                self.created_at,
+                                                self.updated_at,
+                                                self.event,
+                                                [user.id for user in self.users.all()])
     def __str__(self):
         """
         Magic method that returns string representation of
-        event instance object.
+        task instance object.
 
         :return: task title, task description, task status
         """
 
-        return "{} {} {}".format(self.title, self.description, self.status)
+        return "{} {} {} {} {} {} {} {}".format(self.id,
+                                                self.title,
+                                                self.description,
+                                                self.status,
+                                                self.created_at,
+                                                self.updated_at,
+                                                self.event,
+                                                [user.id for user in self.users.all()])

```

Comment:
maybe this field could return enough long output.

Reply:
leave as it is yet